### ISSUE TITLE
Calculating average probe measurement time

### ISSUE THREAD

**Issue by hawkinsw** [network-quality/draft-ietf-ippm-responsiveness#53]
https://github.com/network-quality/draft-ietf-ippm-responsiveness/blob/38f2b241d6f96588488a89c864d023bfa0432075/draft-ietf-ippm-responsiveness.txt#L481

In each probe there are five measurements (obviously this number is subject to change given #52 , etc). However, from the draft, it is not clear whether the "average probe duration" is 

```
total = 0
repeat N times:
  total += (DNS Time + TCP Time + TLS Time + HTTP Time [unsat] + HTTP Time [sat])/5

average probe duration = total / N
```
or

```
total = 0
repeat N times:
  total += (DNS Time + TCP Time + TLS Time + HTTP Time [unsat] + HTTP Time [sat])

average probe duration = total / N
```

**Comment by LPardue** [comment]
+1 to clarifying this text

**Comment by cpaasch** [comment]
Should be addressed with af69ae22ea917a7afd5ade9a2b6d1948d63ac558
